### ISSUE TITLE
[BUG] [Linter] H037 false positive with AlpineJS

### ISSUE THREAD

**Issue by danjac** [Riverside-Healthcare/djLint#639]
- [ x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [ x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [ x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Fedora Linux 38
 - Python Version: 3.11
 - djLint Version: 1.27.1
 - template language: Django

## Issue

I have a number of templates using attributes in addition to equivalent AlpineJS bindings. For example:

```html
<img src="img.jpg" :src="isLoaded ? url : defaultValue" />
```

This is being flagged e.g. **H037 10:8 Duplicate attribute found. src**

Using the full Alpine namespace e.g . `x-bind:src` results in the same error.

This was not an issue with djLint 1.26.

I could add H037 to the ignore list, but I would prefer that djlint flags duplicate issues when they are genuine duplicates.

## How To Reproduce

See above

## Contents of .djlintrc/pyproject.toml [tool.djlint]

```yaml
[tool.djlint]
ignore = "H030,H031,H017,H021"
```

**Comment by khink** [comment]
+1 on this report. We get a false positive for `<tbody class="bg-white" x-data="{ open{{ item.history_id }}: false }" x-bind:class="open{{ item.history_id }} ? 'bg-gray-50' : ''">`

**Comment by christopherpickering** [comment]
Thanks, will get this fixed.
